pycbc/transforms.py

```diff
@@ -612,7 +612,101 @@ def inverse_jacobian(self, maps):
         ref_mass=1.4
         mchirp = maps['mchirp']
         return (2.**(-1./5) * self.ref_mass / mchirp)**(5./6)
+    
+######################################################################################################### TEST ################################################################################################################################################
+import numpy as np
+from pycbc.waveform.waveform_modes import jframe_to_l0frame
+
+class alignTotalSpin(BaseTransform):
+    
+    """Parameters
+    ----------
```

Comment:
This isn't following the numpydoc formatting, and won't display correctly in the online docs. Can you update this, using other code as an example?